Dockerfile

```diff
@@ -1,12 +1,12 @@
 FROM python:2.7
 
 ENV PYTHONPATH=/app
-RUN apt-get update -qq
-RUN apt-get install mysql-client -qq
 
 ADD requirements.txt /requirements.txt
 ADD requirements /requirements
 RUN pip install -r /requirements.txt
+CMD python ./manage.py syncdb --noinput && \
```

Comment:
Should we add back the `docker-compose run --rm web python manage.py createsuperuser` to the README since, we are now doing the syncdb via Docker and are not creating a superuser then?